testing/it/ui-js/src/content/jcr_root/apps/core/wcm/tests/clientlibs/clientlib-base/.content.xml

```diff
@@ -3,4 +3,4 @@
     jcr:primaryType="cq:ClientLibraryFolder"
     allowProxy="{Boolean}true"
     categories="[cmp-test.base]"
-    embed="[cmp-test.grid,core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.pdfviewer.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v2,core.wcm.components.search.v1,core.wcm.components.form.text.v2,core.wcm.components.form.container.v2]"/>
+    embed="[cmp-test.grid,core.wcm.components.accordion.v1,core.wcm.components.tabs.v1,core.wcm.components.carousel.v1,core.wcm.components.pdfviewer.v1,core.wcm.components.image.v2,core.wcm.components.breadcrumb.v3,core.wcm.components.search.v1,core.wcm.components.form.text.v2,core.wcm.components.form.container.v2]"/>
```

Comment:
Is core.wcm.components.breadcrumb.v3 defined or we stick with core.wcm.components.breadcrumb.v2

Reply:
good catch, addressed in https://github.com/adobe/aem-core-wcm-components/pull/1430/commits/930ec977888113d3218047c5eb16ad1f07116c38